pkg/kapp/clusterapply/delete_change.go

```diff
@@ -56,6 +56,11 @@ func (c DeleteChange) IsDoneApplying() (ctlresm.DoneApplyState, []string, error)
 		return ctlresm.DoneApplyState{Done: true, Successful: true, Message: "Resource orphaned"}, nil, nil
 	}
 
+	if res.IsDeleting() && len(res.Finalizers()) > 0 {
+		return ctlresm.DoneApplyState{Done: false, Message: "waiting on finalizers being removed"},
+			[]string{"Resource deletion step is waiting on finalizers being removed"}, nil
```

Comment:
It would be useful if list of remaining finalizers was printed

Reply:
Sure, That's a great input. Will make the required changes to start printing them.